docs/source/simulation_locations.rst

```diff
@@ -14,6 +14,8 @@ Simulations have NERSC HPSS paths at: ::
 
   /home/projects/e3sm/www/WaterCycle/E3SMv2/LR
 
+The preferred retrieval method is ESGF. All the native output is available at `ESGF <https://esgf-node.llnl.gov/search/e3sm/?model_version=2_0>`_.
+
```

Comment:
@forsyth2 the `ensemble_member` maps to the corresponding `0201`. The table Tony made here: https://github.com/E3SM-Project/CMIP6-Metadata/pull/9#issuecomment-1246086256 provide such mapping. for instance, `v2.LR.historical_0201` needs ` ensemble_member=ens3` in the link

Reply:
@chengzhuzhang Thank you for the link! Since the ESGF links are a bit convoluted and it's not clear how what to search for to find a specific simulation, I decided to include the ESGF links in the table.